release/v0.22 <- Glitch2 - Asset Upload - increase timeout

dmitry-yudakov opened the pull request:
#GLI-27

dmitry-yudakov committed:
feat(world-builder): increase timeout for asset upload

#GLI-27

jor-rit reviewed (approved):
Yeah, env not really needed. 
But i'm fine with it this way, might be useful for local development when debugging.

dmitry-yudakov committed:
fix(world-builder): incorrect request for upload asset after closing upload dialog

#GLI-27

worldId is set in the store on init but then the model was reset on dialog closing and the worldId would be lost

dmitry-yudakov commented:
Included also the fix for losing worldId in store.
